x-pack/plugins/ml/server/routes/management.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { wrapError } from '../client/error_wrapper';
+import type { RouteInitialization } from '../types';
+import { listTypeSchema } from './schemas/management_schema';
+
+import { jobServiceProvider } from '../models/job_service';
+import { checksFactory } from '../saved_objects';
+import { BUILT_IN_MODEL_TAG } from '../../common/constants/data_frame_analytics';
+import { BUILT_IN_MODEL_TYPE } from '../../common/constants/trained_models';
+import type {
+  AnomalyDetectionManagementItems,
+  AnalyticsManagementItems,
+  TrainedModelsManagementItems,
+} from '../../common/types/management';
+
+/**
+ * Routes for management service
+ */
+export function managementRoutes({ router, routeGuard }: RouteInitialization) {
+  /**
+   * @apiGroup Management
+   *
+   * @api {get} /api/ml/management/list/:listType Management list
+   * @apiName ManagementList
+   * @apiDescription Returns a list of anomaly detection jobs, data frame analytics jobs or trained models
+   *
+   * @apiSchema (params) listTypeSchema
+   *
+   */
+  router.get(
+    {
+      path: '/api/ml/management/list/{listType}',
+      validate: {
+        params: listTypeSchema,
+      },
+      options: {
+        tags: ['access:ml:canGetJobs'],
```

Comment:
do we need some other tags? as this endpoint also returns models

Reply:
yes, I believe this should be admin create permissions for each type.